app/uk/gov/hmrc/customsservicestatus/models/CustomsServiceStatus.scala

```diff
@@ -32,13 +33,23 @@ case class CustomsServiceStatus(
 )
 
 object CustomsServiceStatus {
+  import scala.jdk.CollectionConverters.ListHasAsScala
 
   val mongoFormat: OFormat[CustomsServiceStatus] = {
     implicit val instantFormat: Format[Instant] = MongoJavatimeFormats.instantFormat
     Json.using[WithDefaultValues].format[CustomsServiceStatus]
   }
 
   implicit val format: OFormat[CustomsServiceStatus] = Json.using[WithDefaultValues].format[CustomsServiceStatus]
+
+  implicit lazy val configLoader: ConfigLoader[List[CustomsServiceStatus]] = ConfigLoader(_.getConfigList).map(
```

Comment:
why are we removing the pureconfig lib and loading config like this ?

Reply:
No need for it mostly. But also it was proving to be difficult to migrate to scala 3

Reply:
hmm, I see the repo is active with regular commits so if its still under v1 it should be fine, looks like they don't release a new version for each commit like other libs do..
it already seems to support scala 3 but if its making our migration difficult, fine to remove it as we dont use it in most of our repos ( I like the lib  as we can directly read config into case classes.. etc )

Reply:
I also like it. The model is so simple though with id, name and description. I would've liked this sort of functionality to be part of either the core libraries or the play framework.

Reply:
I just had another go - and it's still not playing nice. pureconfig.generic.auto._ is no longer available. Another concern I had was that it wasn't possible to mock the values being red form the config files, which is now possible when using the playframework defauled config reader.